kernel/kernel-impl/src/test/java/org/sakaiproject/site/impl/test/SiteAliasCleanupNotificationActionIntegrationTest.java

```diff
@@ -99,7 +99,9 @@ public void disabledTestSiteDeletionTriggersSiteAliasDeletion()
 		}
 		
 		List remainingSiteAliases = aliasService.getAliases(site.getReference());
-		Assert.assertEquals("Expected all site aliases to be deleted on site deletion", 0, 
+		//SAK-31415 soft deletion enabled
+		//Assert.assertEquals("Expected all site aliases to be deleted on site deletion", 0,
+		Assert.assertEquals("Expected all site aliases to remain on soft site deletion", createdSiteAliases.size(),
```

Comment:
The test still works on trunk, the comment is wrong and should also be removed as they are still good and useful, why not just fix the test so it works correctly by removing the site?